Dockerfile.base

```diff
@@ -163,6 +163,7 @@ RUN <<EOF
 	pip install --no-cache-dir \
 		-r https://raw.githubusercontent.com/zephyrproject-rtos/zephyr/main/scripts/requirements.txt \
 		-r https://raw.githubusercontent.com/zephyrproject-rtos/mcuboot/main/scripts/requirements.txt \
+		esptool>=5.0.2 \
```

Comment:
```suggestion
		"esptool>=5.0.2" \
```

Reply:
ok, but why `esptool` need it and `nrf-regtool` doesnt?

Reply:
Because of `>`.